### ISSUE TITLE
Build kotlin2js

### ISSUE THREAD

**Issue by cbeust** [cbeust/kobalt#299]
Project:  https://github.com/Kotlin/kotlinx.html

**Comment by DevCharly** [comment]
seems that @pabl0rg has already worked on a Kobalt build:
https://github.com/pabl0rg/kotlinx.html/commits/build.with.kobalt

**Comment by pabl0rg** [comment]
yeah i did briefly.  The JB folks know and love maven, that may be why -- if i remember correctly -- cy6erGn0m seemed more puzzled than pleased with the PR

**Comment by orangy** [comment]
Well, the build is jvm-only there?

**Comment by pabl0rg** [comment]
Yeah, that too ;-)

**Comment by cbeust** [comment]
Took a quick look, this build uses the [`frontend-maven-plugin`](https://github.com/eirslett/frontend-maven-plugin) which lets you install and manage various Javascript package managers and build tools (npm, grunt, etc...).

Not very hard to implement, we can probably get away with just creating a Kobalt `npm` plug-in.
